zingolib/src/testutils/lightclient.rs

```diff
@@ -88,8 +89,25 @@ pub mod from_inputs {
         crate::data::proposal::ProportionalFeeProposal,
         crate::wallet::propose::ProposeSendError,
     > {
+        // TOdo fix expect
         let request = transaction_request_from_send_inputs(raw_receivers)
             .expect("should be able to create a transaction request as receivers are valid.");
         proposer.propose_send(request).await
     }
 }
+
+/// gets stati for a vec of txids
+pub async fn lookup_stati(
```

Comment:
statuses would be immediately understood, whereas stati, though I figured out what it meant, is not as easily understandable.
```suggestion
pub async fn lookup_statuses(
```

Reply:
I had the exact same problem.